### DISCUSSION TITLE
Add button to randomly enable mods in mod select

### DISCUSSION THREAD

**Discussion by pigeonskew** [ppy/osu#30474] [Ideas]
Got inspired from the idea that there's a random button in song select. 

Considering that we have way more mods now compared to stable, it'd be cool to have a random button that randomly enables mods depending on how many mods you want it to enable (which would be its only custom setting).

Of course, it should know what mods are compatible to each other and which aren't.

**Comment by peppy** [comment]
I like this idea a lot.
